devscripts/generate_fake_sessions.py

```
@@ -0,0 +1,122 @@
+        print("Error: No valid channel or exercise content found.")
+        return None, None
+
+    return channel_result[0], content_result[0]
+
+from kolibri.core.logger.models import ContentSessionLog
+from kolibri.core.auth.models import FacilityUser, Facility
+from django.db import transaction
+
+def main():
+
+    from kolibri.core.content.models import ChannelMetadata, ContentNode
```

Ruff (0.8.2)

`kolibri.core.content.models.ChannelMetadata` imported but unused

Remove unused import: `kolibri.core.content.models.ChannelMetadata`

(F401)